recipes/herepy/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "herepy" %}
+{% set version = "3.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abdullahselek/HerePy/archive/{{ version }}.tar.gz
+  sha256: 514008891b1f2b3e5b49e948a22dfd7e0bb3baddd99182c15525a01c0a227970
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```